tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+    const testPlanId = $('#test_plan_pk').data('testplanPk');
+    const testCases = {
+        'confirmed': [],
+        'not_confirmed': []
+    };
+
+    jsonRPC('TestCase.filter', [{'plan': testPlanId}], function(data) {
+        for (var i = 0; i < data.length; i++) {
+            var testCase = data[i];
+            if (testCase.case_status_id === 2) {
```

Comment:
this may break for any number of reasons, most definitely when the TestCaseStatus model gets removed b/c all of this can be controlled via boolean field or an integer choice field (less ideal). Add a TODO comment above so we don't forget to fix it, see comments above `class TestCaseStatus`